src/content/community/contributor-roles/_index.en.md

```diff
@@ -16,6 +16,8 @@ relevant to our structure or possible with our tooling:**
 * **We hope to eventually move to Kubernetes OWNERS and Prow, but until we do so we
   can't support advanced role-based automation (reviewers vs approvers;
   PR workflow commands like /okay-to-test, /lgtm, /approved).**
+* **Project Owners are given responsibility for some tasks that are handled by dedicated teams in Kubernetes (security responses, Code of
```

Comment:
The ** ^ is not closed, please see the render:

https://deploy-preview-520--elated-bell-2913d9.netlify.app/community/contributor-roles/

Reply:
Good catch, IDK how I missed that one. Thanks for the renders by the way, they are so cool/helpful!